include/gul14/string_view.h

```diff
@@ -316,7 +316,7 @@ class basic_string_view {
             pos = len_ - (pos+1);
         const_reverse_iterator iter = std::find_first_of
                 ( this->crbegin () + pos, this->crend (), s.cbegin (), s.cend (), traits::eq );
-        return iter == this->crend () ? npos : reverse_distance ( this->crbegin (), iter);
```

Comment:
We might consider putting parentheses around the right-hand term of the `? :` operator. I had to look up the precedence to figure out that this is correct. But we also might not... :)

Reply:
If it is unclear we should put parens. Done.